src/elements/content-sidebar/activity-feed/task-form/__tests__/TaskForm-test.js

```diff
@@ -191,6 +192,22 @@ describe('components/ContentSidebar/ActivityFeed/task-form/TaskForm', () => {
         expect(wrapper.find('PillSelectorDropdown').hasClass('scrollable'));
     });
 
+    describe('approver input', () => {
+        test('should show error when approver input is incomplete (no pill selected)', () => {
+            const value = 'not-a-user';
+            const getApproverWithQuery = jest.fn();
+            const wrapper = render({ getApproverWithQuery });
+            let input = wrapper.find('PillSelector[data-testid="task-form-assignee-input"]');
+
+            input.prop('onInput')({ target: { value } });
+            input.prop('onBlur')();
+            wrapper.update();
+
+            input = wrapper.find('PillSelector[data-testid="task-form-assignee-input"]');
+            expect(input.prop('error')).toBe(commonMessages.invalidUserError.defaultMessage); // 'Invalid User'
```

Comment:
I suppose the possible default string is clear from the key name, so the comment might not be needed in this context